CaseStudies/ssp/docs/SRS/SRS.tex

```diff
@@ -128,23 +128,25 @@ \subsection{Table of Units}
 \setlength{\tabcolsep}{20pt}
 \begin{tabular}{  l  l  l  }
 \hline
-\textbf{Physical Property} & \textbf{Name} & \textbf{Symbol} \\
+\textbf{Symbol} & \textbf{Unit} & \textbf{SI} \\
 \hline
-force & Newton & \si{\newton} \\
-length & meter & \si{\meter}  \\
-pressure & Pascal & $\si{\pascal}=\si{\newton\per\square\meter}$ \\
-angle & degree & \si{\degree}  \\
+\si{\newton} & force & Newton \\
+\si{\meter} & length & meter \\
+$\si{\pascal}=\si{\newton\per\square\meter}$ & pressure & Pascal \\
+\si{\degree} & angle & degree  \\
 \hline
 \end{tabular}
 \renewcommand{\arraystretch}{1}
 
+\bmac{How is it decided which units show derivation information?}
+
```

Comment:
I remember we discussed this, but I don't remember what the "rule" was that we came up with.  @bmaclach, can you please remind me.  I can then record this information in the template file for future reference.

Reply:
The rule was that if the unit was derived from only units that were already in the table (i.e. only units that are used in the SRS), then we would show the derivation information. Otherwise we would not.

So for SSP, derivation of pascals as `N/m^2` is shown since `N` and `m` were both already in the table. Derivation of newtons is not shown as `kg` and `s` were not in the table.

Reply:
That's it!  I've added this information to the cas741 blank template for the SRS.  The more rules we can find like this, the more that Drasil can automate and the more that it can check.